src/components/base/InputCurrency/index.js

```diff
@@ -198,6 +208,7 @@ class InputCurrency extends PureComponent<Props, State> {
       <Input
         {...this.props}
         ff="Rubik"
+        ref={this.onRef}
```

Comment:
is it intended to not use `React.createRef` here, like in `BitcoinKind.js`?
in your opinion, should we refacto all our refs with it?